storage/PreloaderThread.cpp

```diff
@@ -55,22 +56,27 @@ void PreloaderThread::run() {
       std::vector<block_id> blocks = relation.getBlocksSnapshot();
       for (block_id current_block_id : blocks) {
         try {
-          BlockReference current_block = storage_manager_->getBlock(current_block_id, relation);
+          BlockReference current_block =
+              storage_manager_->getBlock(current_block_id, relation);
         } catch (...) {
           LOG(ERROR) << "Error after loading " << blocks_loaded << "blocks\n";
           throw;
         }
         ++blocks_loaded;
         if (blocks_loaded == num_slots) {
-          // The buffer pool has filled up. But, some database blocks are not loaded.
-          printf(" The database is larger than the buffer pool. Only %lu blocks were loaded ",
-                 blocks_loaded);
+          // The buffer pool has filled up. But, some database blocks are not
+          // loaded.
+          printf(
+              " The database is larger than the buffer pool. Only %lu blocks "
+              "were loaded", blocks_loaded);
           return;
         }
       }
+      LOG(INFO) << "Relation " << relation.getName()
+                << " completely preloaded in buffer pool\n";
     }
   }
-  printf(" Loaded %lu blocks ", blocks_loaded);
+  printf("Loaded %lu blocks", blocks_loaded);
```

Comment:
Some whitespaces are intended to add.

https://github.com/pivotalsoftware/quickstep/pull/172#discussion_r60004795